contracts/Transfer.sol

```diff
@@ -112,7 +110,8 @@ contract Transfer is FraudProofHelpers {
     function processTx(
         bytes32 _balanceRoot,
         bytes32 _accountsRoot,
-        Types.Transaction memory _tx,
+        bytes memory txs,
+        uint256 i,
```

Comment:
This is the onchain `processTx` that we use in processBatch. It takes `bytes memory txs` and `i`